README.markdown

```diff
@@ -726,6 +735,7 @@ Use access control as the leading property specifier. The only things that shoul
 private let message = "Great Scott!"
 
 class TimeMachine {  
+
```

Comment:
This space feels unnecessary for clarity to me.

Reply:
I also think it's unnecessary and not consistent with the way we edit tutorials and other books.